engine/source/gui/core/guiCanvas.cpp

```diff
@@ -648,17 +665,27 @@ bool GuiCanvas::processInputEvent(const InputEvent* event)
                     else
                         retval = responder->onGamepadButtonPressed(XI_DPAD_UP);
                 }
-                else
+                else if (event->objType == XI_THUMBLX)
                 {
                     if (down)
                         retval = responder->onGamepadButtonPressed(XI_DPAD_RIGHT);
                     else
                         retval = responder->onGamepadButtonPressed(XI_DPAD_LEFT);
                 }
+                else if (event->objType == XI_LEFT_TRIGGER)
+                {
+                    retval = responder->onGamepadButtonPressed(XI_LEFT_TRIGGER);  
+                }
+                else if (event->objType == XI_RIGHT_TRIGGER)
+                {
+                    retval = responder->onGamepadButtonPressed(XI_RIGHT_TRIGGER);  
+                }
                 gLastEventTime = curTime;
             }
 
             return retval;
+        } else {
+            Con::printf("Input fell outside current logic. Maybe theres a bug!");
```

Comment:
If this only occurs when there's an issue, it should use Con::warnf or Con::errorf

Reply:
Gotcha, I'll update this now.